src/openrct2/platform/Platform.Win32.cpp

```diff
@@ -37,6 +37,10 @@
 
 #    include <iterator>
 
+#    if _WIN32_WINNT < 0x600
+#        define swprintf_s(a, b, c, d, ...) swprintf(a, b, c, ##__VA_ARGS__)
```

Comment:
Does `std::swprintf` work? Or rather is it possible to have `std::swprintf` be used instead of `swprintf_s` by default?

Reply:
Is it worth it though?